scripts/react-tester.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-env node */
+
+const { execSync } = require( 'child_process' );
+const { mkdirSync, rmdirSync, copyFileSync } = require( 'fs' );
+const { resolve: resolvePath } = require( 'path' );
+const satisfiesSemver = require( 'semver/functions/satisfies' );
+const semverMinor = require('semver/functions/minor')
+
+const PACKAGE_PATH = resolvePath( __dirname, '..' );
+const TESTS_PATH = resolvePath( PACKAGE_PATH, 'react-tests' );
+
+const packageInfo = require( '../package.json' );
+const availableVersions = getVersions();
+const semverRange = getReactVersion( packageInfo );
+const versionsInRange = getVersionsInRange( semverRange, availableVersions );
+const versionsToTest = getLatestPatches( versionsInRange );
+
+try {
+	console.log( '--- Ultimate CKEditor 4 - React Integration Tester ---' );
+
+	rmdirSyncRecursive( TESTS_PATH );
+	mkdirSync( TESTS_PATH );
+
+	versionsToTest.forEach( testVersion );
+
+	rmdirSyncRecursive( TESTS_PATH );
+	console.log( 'Done without errors. Have a nice day!' );
```

Comment:
Not sure if it's needed :thinking: Maybe assuming that if there is no error message means the run was successful will be enough?

Reply:
I think it's nice to have the end message - you can be sure then that everything was executed, not just that there are no errors. This prevents the situation where for some reason the process was killed in the meantime and you might not notice it.